nginx/config-files/hydroshare-ssl-nginx-ingress.conf.template

```diff
@@ -24,7 +24,11 @@ server {
 server {
     listen          443 ssl;
     server_name     FQDN_OR_IP;
-    root            /system/static/; 
+    root            /system/static/;
```

Comment:
what's this doing in here?  Please check merge conflict resolution, this has nothing to do with this PR and has already been merged.

Reply:
Alva has made some direct commits and they were supposed to be backed out. I'll work with Martin to have this coordinated/reconciled.